.git-blame-ignore-revs

```diff
@@ -0,0 +1,14 @@
+# Fix trailing whitespace and newline at EOF
+483f9a08ca2a05cb60a0e838fec03b465cc56c26
```

Comment:
Won't these commit ids not work after squash?

Reply:
Not planning to squash this PR on purpose.